image-properties: make the tag-name arrays static

They are file-scope and non-static, so the extension exports data symbols
called flash, creator, description, camera_model and 14 more. Any library
already loaded into nautilus that exports the same name wins the lookup:
libncursesw exports a *function* named flash, and it reaches nautilus through
gvfs -> libedit. append_gexiv2_tag then walks ncurses' code as if it were an
array of tag-name pointers and gexiv2_metadata_has_tag dereferences the first
word of it, so opening an image's Properties crashes with SIGSEGV. Not
platform specific; found on ppc64le.

--- a/extensions/image-properties/nautilus-image-properties-model.c
+++ b/extensions/image-properties/nautilus-image-properties-model.c
@@ -36,23 +36,23 @@
 } NautilusImagesPropertiesModel;
 
 /* tags and their alternatives */
-const char *title[] = { "Xmp.dc.title", NULL };
-const char *camera_brand[] = { "Exif.Image.Make", NULL };
-const char *camera_model[] = { "Exif.Image.Model", "Exif.Image.UniqueCameraModel", NULL };
-const char *created_on[] = { "Exif.Photo.DateTimeOriginal", "Xmp.xmp.CreateDate", "Exif.Image.DateTime", NULL };
-const char *exposure_time[] = { "Exif.Photo.ExposureTime", NULL };
-const char *aperture_value[] = { "Exif.Photo.ApertureValue", NULL };
-const char *iso_speed_ratings[] = { "Exif.Photo.ISOSpeedRatings", "Xmp.exifEX.ISOSpeed", NULL };
-const char *flash[] = { "Exif.Photo.Flash", NULL };
-const char *metering_mode[] = { "Exif.Photo.MeteringMode", NULL };
-const char *exposure_mode[] = { "Exif.Photo.ExposureMode", NULL };
-const char *focal_length[] = { "Exif.Photo.FocalLength", NULL };
-const char *software[] = { "Exif.Image.Software", NULL };
-const char *description[] = { "Xmp.dc.description", "Exif.Photo.UserComment", NULL };
-const char *subject[] = { "Xmp.dc.subject", NULL };
-const char *creator[] = { "Xmp.dc.creator", "Exif.Image.Artist", NULL };
-const char *rights[] = { "Xmp.dc.rights", NULL };
-const char *rating[] = { "Xmp.xmp.Rating", NULL };
+static const char *title[] = { "Xmp.dc.title", NULL };
+static const char *camera_brand[] = { "Exif.Image.Make", NULL };
+static const char *camera_model[] = { "Exif.Image.Model", "Exif.Image.UniqueCameraModel", NULL };
+static const char *created_on[] = { "Exif.Photo.DateTimeOriginal", "Xmp.xmp.CreateDate", "Exif.Image.DateTime", NULL };
+static const char *exposure_time[] = { "Exif.Photo.ExposureTime", NULL };
+static const char *aperture_value[] = { "Exif.Photo.ApertureValue", NULL };
+static const char *iso_speed_ratings[] = { "Exif.Photo.ISOSpeedRatings", "Xmp.exifEX.ISOSpeed", NULL };
+static const char *flash[] = { "Exif.Photo.Flash", NULL };
+static const char *metering_mode[] = { "Exif.Photo.MeteringMode", NULL };
+static const char *exposure_mode[] = { "Exif.Photo.ExposureMode", NULL };
+static const char *focal_length[] = { "Exif.Photo.FocalLength", NULL };
+static const char *software[] = { "Exif.Image.Software", NULL };
+static const char *description[] = { "Xmp.dc.description", "Exif.Photo.UserComment", NULL };
+static const char *subject[] = { "Xmp.dc.subject", NULL };
+static const char *creator[] = { "Xmp.dc.creator", "Exif.Image.Artist", NULL };
+static const char *rights[] = { "Xmp.dc.rights", NULL };
+static const char *rating[] = { "Xmp.xmp.Rating", NULL };
 
 static void
 nautilus_images_properties_model_free (NautilusImagesPropertiesModel *self)
